pkg/api/resource/quantity.go

```diff
@@ -502,7 +519,7 @@ func (q *Quantity) RoundUp(scale Scale) bool {
 // Add adds the provide y quantity to the current value. If the current value is zero,
 // the format of the quantity will be updated to the format of y.
 func (q *Quantity) Add(y Quantity) {
-	q.s = nil
+	q.s = ""
```

Comment:
Yes.  We may need a more holistic quantity stress tester (I did one
for add and remove, but we might want to get more formal).